lib/pact/command/cont_command.ex

```diff
@@ -273,6 +307,11 @@ defmodule Kadena.Pact.ContCommand do
 
   @spec sign_commands(signs :: list(), cmd :: json_string_payload(), keypairs()) ::
           valid_sign_commands()
+  defp sign_commands([], cmd, []) do
+    {:ok, sign_command} = Sign.sign(cmd, nil)
+    {:ok, [sign_command]}
+  end
```

Comment:
@Odraxs this code snippet is a bit confusing to me, IMO it can be handled in a better way